learn2track/models/gru.py

```diff
@@ -143,10 +152,19 @@ def load(self, path):
 
     @classmethod
     def create(cls, path, **kwargs):
-        loaddir = pjoin(path, cls.__name__)
-        hyperparams = smartutils.load_dict_from_json_file(pjoin(loaddir, "hyperparams.json"))
-        hyperparams.update(kwargs)
-
-        model = cls(**hyperparams)
-        model.load(path)
+        try:
+            loaddir = pjoin(path, cls.__name__)
+            hyperparams = smartutils.load_dict_from_json_file(pjoin(loaddir, "hyperparams.json"))
+            hyperparams.update(kwargs)
+
+            model = cls(**hyperparams)
+            model.load(path)
+        except FileNotFoundError:
```

Comment:
That means the burden is put on every script that uses experiments. If your approach is working correctly, that would be pretty neat and help cleaning the scripts. Just add a print saying the unfinished model is being used.

But now that I thought about it, when is this happening? That is, you have a model save in training/ but not in the root folder of the experiment?